common/src/main/resources/assets/wynntils/urls.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "version": 2
+    "version": 3
```

Comment:
I did just realize that I did not bump this when I added changelogs. Oops.

Reply:
Should we bump current to 3 now (better late than never?!?), and make it 4 in this PR? I guess if no-one has complained then maybe it does not matter so much now.

Reply:
Let's leave it, I guess. Seems like it doesn't matter and the bundled urls can do the job.